output/components.eslint.txt

```diff
@@ -2,6 +2,10 @@
 
 The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
 
+/home/travis/build/Talend/ui/packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js
+   82:2  error  mountDocumentHandler should be placed after onChange                    react/sort-comp
+  120:2  error  componentWillReceiveProps should be placed before componentWillUnmount  react/sort-comp
```

Comment:
Must be fixed.

Reply:
In fact this is my falt. I wanted to isolate the set of method I wanted to extract (TODO)

Reply:
For now I just suppress them, to keep the blocs